packages/protocol/contracts/identity/IdentityProxy.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.13;
+
+import "../common/ExternalCall.sol";
+
+contract IdentityProxy {
```

Comment:
I am against overloading `Proxy` like this
We have tooling all over the place which assumes this suffix implies OZ unstructured storage proxy pattern
Is there a synonym we could use or a different naming scheme?

Reply:
`Forwarder`? `Relayer`? Maybe a little more abstract, but `IdentityAlias` if we keep the semantics of using it in the identity scheme in the name.

Reply:
I like `Relay` personally but basically anything other than `Proxy` is good imo